lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -229,7 +226,7 @@ private JourneyResponse getJourneyResponse(
                 LOGGER.info(
                         LogHelper.buildLogMessage(
                                 "resetIdentity flag is enabled, reset users identity."));
-                return JOURNEY_RESET_SESSION_IDENTITY;
+                return JOURNEY_IPV_GPG45_MEDIUM;
```

Comment:
Sorry @sammyiyke  - this needs to go to a slightly different place. We still need to display the extra reprove identity screen, so it needs to go to `NEW_P2_IDENTITY`/`REPROVE_IDENTITY` (rather than `NEW_P2_IDENTITY`/`START`)